### PR TITLE
autoprefixer: Add errorIfNotFound option

### PR DESCRIPTION
**Pull request by papandreou**
Defaults to true to match previous behavior

### CODE REVIEW

File: lib/transforms/autoprefixer.js [hunk 1]
```diff
@@ -17,13 +17,17 @@ module.exports = (
...
-            cssAssets.length
-          } css asset(s) while --browsers option is active, ` +
-          `but no autoprefixer module is available. Please use npm to install autoprefixer in your project so ` +
-          `the autoprefixer transform can require it.\n${e.message}`;
-        throw e;
+        if (errorIfNotFound) {
+          e.message =
+            `autoprefixer transform: Found ${
+              cssAssets.length
+            } css asset(s) while --browsers option is active, ` +
```
Comment (Munter) [inline lib/transforms/autoprefixer.js]: It's weird that we reference `--browsers` here, in a transform in a project where the concept doesn't even exist.

File: lib/transforms/autoprefixer.js [hunk 2]
```diff
@@ -17,13 +17,17 @@ module.exports = (
...
+            `but no autoprefixer module is available. Please use npm to install autoprefixer in your project so ` +
+            `the autoprefixer transform can require it.\n${e.message}`;
+          throw e;
```
Comment (Munter) [inline lib/transforms/autoprefixer.js]: Isn't the convention in other transforms where we do optional work that we emit a warning instead?

File: lib/transforms/autoprefixer.js (hunk 1 above)
Comment (papandreou) [reply lib/transforms/autoprefixer.js]: Good point, fixed in 300c99b162ac9363df54f671890259b31a7195f4

**Comment by papandreou** [comment]
Release in 5.8.0.

**Pull request merged**